### PR TITLE
fuzz: cleanups for versionbits fuzzer

### PR DESCRIPTION
**Pull request by ajtowns**
Followups for #21380, shouldn't change coverage. Marking as draft to avoid introducing conflicts for the speedy trial PRs.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
cr ACK aa7f418fe32b3ec53285693a7731decd99be4528

File: src/test/fuzz/versionbits.cpp [hunk 1]
```diff
@@ -203,7 +208,7 @@ FUZZ_TARGET_INIT(versionbits, initialize)
         }
 
         // don't risk exceeding max_blocks or times may wrap around
-        if (blocks.size() + period*2 > max_blocks) break;
+        if (blocks.size() + 2 * period > max_blocks) break;
     }
     // NOTE: fuzzed_data_provider may be fully consumed at this point and should not be used further
```
Comment (MarcoFalke) [inline src/test/fuzz/versionbits.cpp:213]: https://github.com/bitcoin/bitcoin/pull/21380#discussion_r595114366 ;)

Comment (ajtowns) [reply src/test/fuzz/versionbits.cpp:213]: The way it currently is makes way more sense to me; I had a go the other way to see if it'd make better use of fuzz magic somehow, but it doesn't; so sticking with what I like.

**Comment by practicalswift** [comment]
Tested ACK aa7f418fe32b3ec53285693a7731decd99be4528

**Comment by MarcoFalke** [comment]
Backported in #21614

**Pull request merged**